images/php-fpm/entrypoints/50-ssmtp.sh

```diff
@@ -19,30 +19,16 @@ fi
 
 if [ ${SSMTP_MAILHUB+x} ]; then
   echo -e "\nmailhub=${SSMTP_MAILHUB}" >> /etc/ssmtp/ssmtp.conf
-else
-  # check if we find a mailhog on 172.17.0.1:1025
-  if nc -z -w 1 172.17.0.1 1025 &> /dev/null; then
-    echo -e "\nmailhub=172.17.0.1:1025" >> /etc/ssmtp/ssmtp.conf
-    return
-  fi
-  # check if mxout.lagoon.svc can do smtp TLS
-  if nc -z -w 1 mxout.lagoon.svc 465 &> /dev/null; then
-    echo -e "UseTLS=Yes\nmailhub=mxout.lagoon.svc:465" >> /etc/ssmtp/ssmtp.conf
-    return
-  fi
-  # Fallback: check if mxout.lagoon.svc can do regular 25 smtp
-  if nc -z -w 1 mxout.lagoon.svc 25 &> /dev/null; then
-    echo -e "\nmailhub=mxout.lagoon.svc:25" >> /etc/ssmtp/ssmtp.conf
-    return
-  fi
-  # check if mxout.default.svc can do smtp TLS
-  if nc -z -w 1 mxout.default.svc 465 &> /dev/null; then
-    echo -e "UseTLS=Yes\nmailhub=mxout.default.svc:465" >> /etc/ssmtp/ssmtp.conf
-    return
-  fi
-  # Fallback: check if mxout.default.svc can do regular 25 smtp
-  if nc -z -w 1 mxout.default.svc 25 &> /dev/null; then
-    echo -e "\nmailhub=mxout.default.svc:25" >> /etc/ssmtp/ssmtp.conf
-    return
-  fi
-fi
+# check if we find a mailhog on 172.17.0.1:1025
+elif nc -z -w 1 172.17.0.1 1025 &> /dev/null; then
+  echo -e "\nmailhub=172.17.0.1:1025" >> /etc/ssmtp/ssmtp.conf
+  return
+# check if mxout.lagoon.svc can do smtp TLS
+elif nc -z -w 1 mxout.lagoon.svc 465 &> /dev/null; then
+  echo -e "UseTLS=Yes\nmailhub=mxout.lagoon.svc:465" >> /etc/ssmtp/ssmtp.conf
```

Comment:
let's make this the fallback if this is Lagoon, as the current default of `mailhub=mxout.lagoon.svc:25` doesn't work on any of the major cloud providers, hence this would actually not solve the problem we're currently facing :)

Reply:
Updated as requested